runtime/tests/integration_tests.rs

```diff
@@ -191,12 +191,40 @@ fn test_fips_cmd_api() {
     let mut model = run_rom_test("mbox");
     let expected_err = Err(ModelError::MailboxCmdFailed(0x000E0006));
 
+    model
+        .soc_ifc()
+        .cptra_fw_rev_id()
+        .at(0)
+        .write(|_| b'R'.into());
+
+    model
+        .soc_ifc()
+        .cptra_fw_rev_id()
+        .at(1)
+        .write(|_| b'A'.into());
+
```

Comment:
The REV_ID registers should be read-only from the SoC side, so this won't work on FPGA/verilator.

Reply:
I will add conditional compilator for emulated model only then,

Reply:
I would recommend just checking that these values are 0 in this PR. Then once we implement the version registers, we can update the test to check them.